src/ensembl/src/shared/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import styles from './Tabs.scss';
+
+export type Tab = {
+  title: string;
+  isDisabled?: boolean;
+};
+
+export type TabsProps = {
+  tabs: Tab[];
+  classNames?: {
+    default?: string;
+    disabled?: string;
+    selected?: string;
+    tabsContainer?: string;
+  };
+  selectedTab: string;
+  selectTab: (selectedTab: string) => void;
+};
+
+export const Tabs = (props: TabsProps) => {
+  const [selectedTab, setSelectedTab] = useState(props.selectedTab);
+
+  const getTabClassNames = (tab: Tab) => {
+    const defaultClassNames = classNames(props.classNames?.default, styles.tab);
+
+    const disabledClassNames = classNames(
+      props.classNames?.disabled,
+      styles.disabled
+    );
+
+    const selectedClassNames = classNames(
+      props.classNames?.selected,
+      styles.selected
+    );
+
+    return classNames(defaultClassNames, {
+      [disabledClassNames]: tab.isDisabled,
+      [selectedClassNames]: tab.title === selectedTab
+    });
+  };
+
+  const tabsContainerClassName = classNames(
+    styles.tabsContainer,
+    props.classNames?.tabsContainer
+  );
+
+  const onTabSelect = (tabTitle: string) => {
+    setSelectedTab(tabTitle);
+    props.selectTab(tabTitle);
+  };
+
+  return (
+    <div className={tabsContainerClassName}>
+      {Object.values(props.tabs).map((tab: Tab) => (
+        <span
+          className={getTabClassNames(tab)}
+          key={tab.title}
+          onClick={tab.isDisabled ? () => null : () => onTabSelect(tab.title)}
```

Comment:
`() => null` can be `noop` from `lodash`? Or you can do an early return inside `onTabSelect` itself. I think we've done the latter in other parts of the site.

Reply:
Done using noop